frontend/src/components/quiz/QuizDialog.tsx

```diff
@@ -177,7 +175,7 @@ const QuizDialogWizard: FC<QuizDialogWizardProps> = ({ onClose }) => {
                   data-cy="progress-bar"
                   className="my-2"
                 />
-                <p id="progress-label" className="m-0 text-center">
+                <p id="progress-label" className="my-4 text-sm text-black text-center">
```

Comment:
There is not need to use text-black. Default html body color is good.

```suggestion
                <p id="progress-label" className="mb-1 mt-4 text-center text-sm">
```

Reply:
Fixed